app/transformations/UserAnswersToETMP/API1830.scala

```diff
@@ -72,11 +84,23 @@ object API1830 extends Transformer {
         (pathIndividualMemberDetails \ Symbol("nino")).json.copyFrom((__ \ Symbol("membersDetails") \ Symbol("nino")).json.pick) and
         (__ \ Symbol("eventType")).json.put(JsString(s"Event${eventType}")) and
         (pathPaymentDetails \ Symbol("amountCrystalised")).json.copyFrom((pathAmountCrystallisedAndDateDetails \ Symbol("amountCrystallised")).json.pick) and
-        (pathPaymentDetails \ Symbol("typeOfProtection")).json.copyFrom(readsTypeOfProtection) and
+        (pathPaymentDetails \ Symbol("typeOfProtection")).json.copyFrom(readsTypeOfProtectionEvent6) and
         (pathPaymentDetails \ Symbol("eventDate")).json.copyFrom((pathAmountCrystallisedAndDateDetails \ Symbol("crystallisedDate")).json.pick) and
         (pathPaymentDetails \ Symbol("freeText")).json.copyFrom((__ \ Symbol("inputProtectionType")).json.pick)).reduce
   }
 
+  private def readsIndividualMemberDetailsEvent8(eventType: EventType): Reads[JsObject] = {
+    (
+      (pathIndividualMemberDetails \ Symbol("firstName")).json.copyFrom((__ \ Symbol("membersDetails") \ Symbol("firstName")).json.pick) and
```

Comment:
I think the memberDetails reads could be farmed off into a val and reused for both event 8 and other event